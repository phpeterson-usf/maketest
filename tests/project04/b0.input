-h $(repo_dir)/b0.hex $(TESTS_DIR)/b0.s
