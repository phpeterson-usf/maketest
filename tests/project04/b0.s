main:
    add r0, r1, r2
    b foo
    add r1, r2, r0
foo:
    add r3, r2, r1
    bx lr
