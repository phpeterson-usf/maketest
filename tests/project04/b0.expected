v2.0 raw
e0810002
ea000000
e0821000
e0823001
e12fff1e
